meta-cube/recipes-core/lxc/files/lxc-overlayscan

```diff
@@ -38,6 +38,11 @@ if [ $(ls -Al "$2"|wc -l) -eq 1 ]; then
 fi
 }
 
+#0.Check flag file
+if [ ! -e /var/lib/lxc/need_scan_duplicate ]; then
```

Comment:
Can you add a more descriptive comment on why we exit if this flag file is present ?